packages/core/src/plugins/stream/SoundcloudPlugin.ts

```diff
@@ -29,7 +30,8 @@ class SoundcloudPlugin extends StreamProviderPlugin {
   async search(query: StreamQuery): Promise<StreamData[] | undefined> {
     const terms = query.artist + ' ' + query.track;
     try {
-      const results = await(await Soundcloud.soundcloudSearch(terms)).json();
+      const results = [];
```

Comment:
The new soundcloud track details api returns single results, rather than an array. We could make a track details call for each search result, this might end up in a lot of network requests though.

Reply:
There should be one search request, then we just display what we can.